config/clouddriver.yml

```diff
@@ -75,9 +75,7 @@ dockerRegistry:
     - name: ${providers.dockerRegistry.primaryCredentials.name}
       address: ${providers.dockerRegistry.primaryCredentials.address}
       username: ${providers.dockerRegistry.primaryCredentials.username:}
-      passwordFile: ${providers.dockerRegistry.primaryCredentials.passwordFile:}
-      repositories: 
-        - ${providers.dockerRegistry.primaryCredentials.repository}
+      passwordFile: ${providers.dockerRegistry.primaryCredentials.passwordFile}
```

Comment:
The motivation to remove the repos field is that GCR now supports `/_catalog` (reading all owned repos of the authenticated user).